src/aap_eda/settings/default.py

```diff
@@ -617,6 +617,13 @@ def get_rulebook_process_log_level() -> RulebookProcessLogLevel:
 DIRECT_SHARED_RESOURCE_MANAGEMENT_ENABLED = settings.get(
     "DIRECT_SHARED_RESOURCE_MANAGEMENT_ENABLED", False
 )
+if DIRECT_SHARED_RESOURCE_MANAGEMENT_ENABLED:
```

Comment:
@bzwei Can we check if the required parameters are set after DIRECT_SHARED_RESOURCE_MANAGEMENT_ENABLED is set. Since those other parameters like

1. RESOURCE_SERVER
2. RESOURCE_JWT_USER_ID
Should be set otherwise we have a bad configuration and we should stop the startup process.